doc/mgr/dashboard.rst

```diff
@@ -347,7 +340,7 @@ You can now access the dashboard using your (JavaScript-enabled) web browser, by
 pointing it to any of the host names or IP addresses and the selected TCP port
 where a manager instance is running: e.g., ``http(s)://<$IP>:<$PORT>/``.
 
-You should then be greeted by the dashboard login page, requesting your
+You should greeted by the dashboard login page, requesting a
```

Comment:
The dashboard login page displays and requests...

Reply:
ack